<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TariffAdminWindow</class>
 <widget class="QWidget" name="TariffAdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>982</width>
     <height>628</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/tariffFrame.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>934</x>
     <y>14</y>
     <width>17</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px; </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/swapIcon.png</normaloff>:/swapIcon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>17</width>
     <height>17</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>356</x>
     <y>22</y>
     <width>126</width>
     <height>37</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px; </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/stationsButton2New.png</normaloff>:/stationsButton2New.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>126</width>
     <height>37</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>22</y>
     <width>126</width>
     <height>37</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px; </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/homeButton2_New.png</normaloff>:/homeButton2_New.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>126</width>
     <height>37</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>958</x>
     <y>14</y>
     <width>17</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px; </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/closeIcon.png</normaloff>:/closeIcon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>17</width>
     <height>17</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>648</x>
     <y>22</y>
     <width>126</width>
     <height>37</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px; </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/trainButton2New.png</normaloff>:/trainButton2New.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>126</width>
     <height>37</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>498</x>
     <y>22</y>
     <width>134</width>
     <height>37</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px; </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/routeButton2New.png</normaloff>:/routeButton2New.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>134</width>
     <height>37</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_11">
   <property name="geometry">
    <rect>
     <x>643</x>
     <y>419</y>
     <width>221</width>
     <height>82</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px; </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/removeAdminButton.png</normaloff>:/removeAdminButton.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>221</width>
     <height>82</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>381</x>
     <y>419</y>
     <width>240</width>
     <height>82</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px; </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/changeAdminButton.png</normaloff>:/changeAdminButton.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>240</width>
     <height>82</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>419</y>
     <width>240</width>
     <height>82</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px; </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/appendAdminButton.png</normaloff>:/appendAdminButton.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>240</width>
     <height>82</height>
    </size>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>115</y>
     <width>890</width>
     <height>280</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 3px solid #6E93FC; border-radius: 5px; background-color: white; </string>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
